auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
there is no mention of it in v9 doc: https://www.odoo.com/documentation/9.0/reference/testing.html